google/cloud/spanner/database_admin_connection_test.cc

```diff
@@ -436,55 +417,169 @@ TEST(DatabaseAdminConnectionTest, UpdateDatabaseGetOperationError) {
 
 /// @test Verify that we can list databases in multiple pages.
 TEST(DatabaseAdminConnectionTest, ListDatabases) {
-  auto mock = std::make_shared<MockDatabaseAdminStub>();
-  Instance in("test-project", "test-instance");
+  constexpr char const* kDatabaseText[5] = {
```

Comment:
nit: here and below, I think east const spelling would be `char const* constexpr` ?

Reply:
```
google/cloud/spanner/database_admin_connection_test.cc:420:15: error: expected unqualified-id                                                                   
  char const* constexpr kDatabaseText[5] = {
              ^
```

Reply:
TIL, thanks for checking.